code/src/App.js

```diff
@@ -1,12 +1,21 @@
 import React from 'react'
 import data from './data.json'
 
-console.log(data)
+import { Header } from "components/Header";
+import { Album } from "components/Album";
+
 
 export const App = () => {
   return (
     <div>
-      Find me in src/app.js!
+      <Header />
+      <section className='album-container'>
+        {data.albums.items.map((album) => { 
+            return ( 
+              <Album key={album.id} albumDetails={album} />
```

Comment:
great job with managing to get the map to work outside the component! maybe you could teach us about the how to work with getting the key in the right place? I think we missed that part! :)